### PR TITLE
Added information about deploying Django projects on AWS and added new file structure for Development_Process for submissions

### PR DESCRIPTION
**Pull request by tsh4fiq**
- Added information about deploying Django projects on AWS, Railway, and Vercel
- Added a new file structure named ./Topics/Development_Process for submission files
- Linked to new information in ./Topics/Development_Process.md
- Added title of submission to README.md

### CODE REVIEW

File: Topics/Development_Process/Django_Deployment_AWS_Railway_Vercel.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ It is highly suggested you read through the following table carefully to learn w
...
-| Price of Storage (Price Charged for Hosting Storage) | <ul><li>Completely free</li></ul> | <ul><li>Only enough to store code behind Django</li><li>No storage for SQLite Database and media files</li></ul> | Free until 100GB of total storage |
+| Price of Storage (Price Charged for Hosting Storage) | <ul><li>Completely free</li></ul> | <ul><li>Only enough to store code behind Django</li><li>No storage for SQLite Database and media files</li></ul> | <ul><li>Free until 100GB of total storage</li></ul> |
 | Security | <ul><li>Minimal security</li><li>SSL certificate to communicate over HTTPS requires a paid for domain</li><li>Process to obtain SSL certificate with domain can be daunting</li></ul> | <ul><li>As Vercel own their own domain, all hosted services communicate over HTTPS for free</li></ul> | <ul><li>As Vercel own their own domain, all hosted services communicate over HTTPS for free</li></ul> |
```
Comment (MicheleMassa802) [inline Topics/Development_Process/Django_Deployment_AWS_Railway_Vercel.md:40]: Vercel block on security duplicated. Please check!

**Comment by MicheleMassa802** [comment]
Overall an excellent looking submission. It's more than an overview of the topic and more like an actual complete report/article on how the deployment of a Django project on these platforms works. 

The table section discussing the pros and cons of each of the 3 deployment options you have chosen is great and easy to understand. However, I have to point out that in the section of the table on security, you seem to have duplicated the Vercel information for both Vercel and Railway (I include a screenshot of it here and a comment on the table section): 
image

All video links seem to be working (they take you to the correct resource) and I appreciate the inline links you have included for basically every section that contain information on the setup and requirements for the specific action you are talking about. It makes the contribution feel very complete and like a one-stop-shop for everything you need on the topic of deployment through Vercel, AWS, and Railway.

Having such a long article, I also appreciate how you included links to each at the top of the file, and how you have made it its own file within a broader directory, which helps keep the structure organized. You have also appropriately updated the readme.

Also, I need you to resolve the conflicts that you now have with main. A PR got merged that changed the structure of Development_Process.md and now you just have to shift your contribution someplace else in the file.

Apart from that, its a very complete and extremely helpful contribution that is very valuable to any team working with Django. Review what I have mentioned here, and I'll merge it.

File: Topics/Development_Process/Django_Deployment_AWS_Railway_Vercel.md (hunk 1 above)
Comment (tsh4fiq) [reply Topics/Development_Process/Django_Deployment_AWS_Railway_Vercel.md]: Edited. Thanks Michele!

**Comment by tsh4fiq** [comment]
Fixed the mistake within the Security row of the Advantages vs. Disadvantages table and resolved merge conflicts. Thanks for the detailed feedback Michele!

**Pull request merged**